app/src/test/resources/com/sefford/artdrian/data/dto/deserializers/index.json

```diff
@@ -0,0 +1 @@
jpg\",imageObjectPosition:\"17% 50%\",imageUnoptimized:!0,imageQuality:100,tags:[\"4k-ready\"]};function m(e){let{Image:t}=e.components||{};return t||j(\"Image\",!0),(0,a.jsx)(\"div\",{style:{width:800},children:(0,a.jsx)(\"div\",{className:\"imageContainer\",children:(0,a.jsx)(t,{className:\"image\",alt:\"2019 Year in Review\",src:\"/static/images/ocean.png\",layout:\"fill\",priority:!0})})})}function b(e={}){let{wrapper:t}=e.components||{};return t?(0,a.jsx)(t,Object.assign({},e,{children:(0,a.jsx)(m,e)})):m(e)}var y=b;function j(e,t){throw new Error(\"Expected \"+(t?\"component\":\"object\")+\" `\"+e+\"` to be defined: you likely forgot to import, pass, or provide it.\")}return v(M);})();\n;return Component;"},"_id":"wallpapers/ghost_waves_002.mdx","_raw":{"sourceFilePath":"wallpapers/ghost_waves_002.mdx","sourceFileName":"ghost_waves_002.mdx","sourceFileDir":"wallpapers","contentType":"mdx","flattenedPath":"wallpapers/ghost_waves_002"},"type":"Wallpaper","slug":"ghost_waves_002","totalDownloads":1528},{"title":"Ghost waves","edition":"001","tags":["4k-ready"],"publishedAt":"2022-08-01T00:00:00.000Z","image":"/static/images/wallpaper/ghost_waves_001_web_preview.jpg","imageQuality":100,"imageUnoptimized":true,"imageObjectPosition":"70% 50%","imageExtension":"png","description":"","className":"bg-wallpaper-ghost_waves_001 hover:bg-wallpaper-ghost_waves_001","body":{"raw":"","code":"var Component=(()=>{var _=Object.create;var n=Object.defineProperty;var c=Object.getOwnPropertyDescriptor;var l=Object.getOwnPropertyNames;var u=Object.getPrototypeOf,w=Object.prototype.hasOwnProperty;var d=(e,t)=>()=>(t||e((t={exports:{}}).exports,t),t.exports),h=(e,t)=>{for(var a in t)n(e,a,{get:t[a],enumerable:!0})},o=(e,t,a,r)=>{if(t&&typeof t==\"object\"||typeof t==\"function\")for(let i of l(t))!w.call(e,i)&&i!==a&&n(e,i,{get:()=>t[i],enumerable:!(r=c(t,i))||r.enumerable});return e};var x=(e,t,a)=>(a=e!=null?_(u(e)):{},o(t||!e||!e.__esModule?n(a,\"default\",{value:e,enumerable:!0}):a,e)),b=e=>o(n({},\"__esModule\",{value:!0}),e);var m=d((C,g)=>{g.exports=_jsx_runtime});var y={};h(y,{default:()=>f,frontmatter:()=>j});var s=x(m()),j={title:\"Ghost waves\",edition:\"001\",publishedAt:\"2022-08-01\",description:\"\",className:\"bg-wallpaper-ghost_waves_001 hover:bg-wallpaper-ghost_waves_001\",image:\"/static/images/wallpaper/ghost_waves_001_web_preview.jpg\",imageObjectPosition:\"70% 50%\",imageUnoptimized:!0,imageQuality:100,tags:[\"4k-ready\"]};function p(e){return(0,s.jsx)(s.Fragment,{})}function v(e={}){let{wrapper:t}=e.components||{};return t?(0,s.jsx)(t,Object.assign({},e,{children:(0,s.jsx)(p,e)})):p(e)}var f=v;return b(y);})();\n;return Component;"},"_id":"wallpapers/ghost_waves_001.mdx","_raw":{"sourceFilePath":"wallpapers/ghost_waves_001.mdx","sourceFileName":"ghost_waves_001.mdx","sourceFileDir":"wallpapers","contentType":"mdx","flattenedPath":"wallpapers/ghost_waves_001"},"type":"Wallpaper","slug":"ghost_waves_001","totalDownloads":992}]},"__N_SSG":true}
```

Comment:
As we are not using the body Javascript element for anything, maybe we can just edit out all this things.

Also, as we are using it as a testing fixture, it's always nice to have the JSONs beautified so we can have human readability. You know, Github does not charge us by line, at this level, I think. :stuck_out_tongue: 

```suggestion
{
  "pageProps": {
    "wallpaper": {
      "title": "Ghost waves",
      "edition": "004",
      "tags": [
        "6k-ready"
      ],
      "publishedAt": "2024-04-21T00:00:00.000Z",
      "image": "/static/images/wallpaper/ghost_waves_004_web_preview.jpg",
      "imageQuality": 100,
      "imageUnoptimized": true,
      "imageObjectPosition": "17% 50%",
      "imageExtension": "jpg",
      "description": "",
      "className": "bg-wallpaper-ghost_waves_004 hover:bg-wallpaper-ghost_waves_004",
      "body": {
        "raw": "",
        "code": ""
      },
      "_id": "wallpapers/ghost_waves_004.mdx",
      "_raw": {
        "sourceFilePath": "wallpapers/ghost_waves_004.mdx",
        "sourceFileName": "ghost_waves_004.mdx",
        "sourceFileDir": "wallpapers",
        "contentType": "mdx",
        "flattenedPath": "wallpapers/ghost_waves_004"
      },
      "type": "Wallpaper",
      "slug": "ghost_waves_004",
      "totalDownloads": 29
    }
  },
  "__N_SSG": true
}
```